Demo/Demo/ViewModels/PayPalViewModel.swift

```diff
@@ -12,8 +12,7 @@ class PayPalViewModel: ObservableObject, PayPalDelegate {
         case mainContent(title: String, content: String, flowComplete: Bool)
     }
 
-    @Published private(set) var state = State.initial
-
+    @Published private(set) var state = State.mainContent(title: "test", content: "test", flowComplete: true)
```

Comment:
Right now this text shows when loading the view initially. Should this be updated to empty strings? Seems odd to have "test test" show in the Demo app.

Reply:
this wasnt supposed to be pushed lol, i was using it to see the `Preview` of the layout in `XCode`